packages/proxy/schema/secrets.ts

```diff
@@ -0,0 +1,74 @@
+import { z } from "zod";
+
+export const BaseMetadataSchema = z
```

Comment:
This one doesn't need to be exported? Also what do you think the naming convention for schema objects should be? I have been declaring them like normal variables, so this would be `baseMetadataSchema`, but I can see the argument for naming them like types because they're basically encoding type information.

Reply:
Yeah I personally like capitalizing global constants and types, and this is certainly the former and arguably the latter, so prefer capitalizing them.